ambari-server/src/main/resources/common-services/ZOOKEEPER/3.4.5/package/scripts/params_linux.py

```diff
@@ -82,6 +82,7 @@
 zoo_cfg_properties_map_length = len(zoo_cfg_properties_map)
 
 zk_principal_name = default("/configurations/zookeeper-env/zookeeper_principal_name", "zookeeper@EXAMPLE.COM")
```

Comment:
I know this was not changed in this patch, however the default value is incorrect. It should be "`zookeeper/_HOST@EXAMPLE.COM`".    How does this affect the `split` in the next line.  I assume `zk_principal_user` would wind up being "zookeeper/_HOST@EXAMPLE.COM" rather than "zookeeper".  This this should be unlikely since "/configurations/zookeeper-env/zookeeper_principal_name" should be set appropriately.

Reply:
good catch, fixed